release/smoke-tests/data-prepper-tar/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+function copy_if_needed() {
+    local SOURCE_DIR=$1
+    local TARGET_DIR=$2
+    local FILE_NAME=$3
+
+    echo "Checking if file exists: ${TARGET_DIR}/${FILE_NAME}"
+
+    if [ -f "${TARGET_DIR}/${FILE_NAME}" ]; then
+        echo "File already exists, skipping copy"
+    else
+        echo "File missing, starting copy"
+        cp "${SOURCE_DIR}/${FILE_NAME}" "${TARGET_DIR}"
+    fi
+}
+
+
+copy_if_needed "${REPO_DIR}/release/archives/linux/build/distributions" "${DOCKER_FILE_DIR}" "${TAR_FILE}"
```

Comment:
This path may need to be variable.

When running the smoke tests against a locally built artifact, this will work.

But, we also want to run smoke tests against artifacts downloaded from elsewhere. So it may make sense to make the source directory configurable.

Reply:
I suppose alternatively, we could download files into that specific directory as a requirement.